### PR TITLE
lwutil refactorings

### PR DESCRIPTION
**Pull request by kbara**
- Centralize is_ipvN_fragment functions
- remove dead code
- use core.lib's [hn]to[nh][ls], now that it is unsigned.

### CODE REVIEW

**Comment by kbara** [comment]
@dpino I've now merged #460 into this patch (which it requires); it needs review before I can merge it into the lwaftr.

File: src/apps/lwaftr/lwutil.lua [hunk 1]
```diff
@@ -66,10 +63,28 @@ local n_ethertype_ipv6 = constants.n_ethertype_ipv6
...
 end
+
 function is_ipv4(pkt)
    return rd16(pkt.data + o_ethernet_ethertype) == n_ethertype_ipv4
 end
 
+function is_ipv6_fragment(pkt)
+   if not is_ipv6(pkt) then return false end
+   return pkt.data[ehs + constants.o_ipv6_next_header] ==
+   constants_ipv6_frag
```
Comment (dpino) [inline src/apps/lwaftr/lwutil.lua]: Would it be possible to move this line up? Makes it more readable and it's still within the 80-column wrap.

**Review by dpino — APPROVED** [review]
LGTM with nit

**Comment by kbara** [comment]
Nit fixed, will merge when tests pass.

**Pull request merged**